sktime/utils/validation/tests/test_check_window_length.py

```diff
@@ -1,10 +1,31 @@
 #!/usr/bin/env python3 -u
 # -*- coding: utf-8 -*-
+import pytest
 
-from sktime.utils.validation.__init__ import check_window_length
+from sktime.utils.validation import check_window_length
 
 
-def test_check_window_length(window_length):
-    assert check_window_length(0.2) == 1  # true
-    assert check_window_length(43) == 43  # True
+def test_check_window_length():
+    # assert check_window_length(0.2) == 1  # true
+    # assert check_window_length(43) == 43  # True
     assert check_window_length(None) is None  # True
+
+
+def test_raises_exception_on_string_arg():
```

Comment:
Take a look at `pytest.mark.parametrize()` to see how you can further simplify these tests

Something like this should work:
```python
@pytest.mark.parametrize("arg", ["str", -1, -0.1])"
def test_window_length_bad_arg(arg):
    ...
```

Reply:
Thats interesting. I would take a look at it